README.md

```diff
@@ -19,3 +19,73 @@ Although this code has been processed successfully on a computer system at the U
 [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/)
 
 IP-117068
+
+## Usage
```

Comment:
would you plz merge `readme` branch into this one?

Reply:
merged